### ISSUE TITLE
What is the difference between using a List-Wise loss and a Pair-Wise loss with a lambda weight function ?

### ISSUE THREAD

**Issue by fangkuann** [tensorflow/ranking#83]
As the title said, what is the difference between using a List-Wise loss directly, like 

`_LOSS = tfr.losses.RankingLossKey.APPROX_NDCG_LOSS`

`loss_fn = tfr.losses.make_loss_fn(_LOSS)`

or use a Pair-Wise loss, with a lambda weight function, like 

`     
_LOSS = tfr.losses.RankingLossKey.PAIRWISE_LOGISTIC_LOSS`

`loss_fn = tfr.losses.make_loss_fn(_LOSS, lambda_weight=ranking_losses.create_ndcg_lambda_weight())`

as Lambda Weight is also an approximation way to optimize NDCG metric

**Comment by bendersky** [comment]
For APPROX_NDCG_LOSS see [this paper](https://ai.google/research/pubs/pub48168). This is basically a loss that optimizes an approximation of the NDCG metric.

For PAIRWISE_LOGISTIC_LOSS, you're optimizing the pairwise logistic loss, but the weights of the examples is dependent on their position in the list, so some information of the list is encoded through weighting, but the optimization is pairwise. See [this paper](https://ai.google/research/pubs/pub47258) on more information.

**Comment by fangkuann** [comment]
Thanks, I will go through these two papers later. And it seems that most learning to rank libs, like XGBoost, LightGBM, only supports the second fashion.
